instrumentation/akka/akka-http-10.0/javaagent/src/test/scala/io/opentelemetry/javaagent/instrumentation/akkahttp/AkkaHttpServerInstrumentationTestSync.scala

```diff
@@ -25,4 +22,12 @@ class AkkaHttpServerInstrumentationTestSync
 
   override protected def stopServer(server: Object): Unit =
     AkkaHttpTestSyncWebServer.stop()
+
+  override protected def configure(
+      options: HttpServerTestOptions
+  ): Unit = {
+    super.configure(options)
+    // FIXME: latest deps does not fill http.status_code
```

Comment:
Should we add an issue for that?

Reply:
To be honest I don't know what the best way to solve this is. Should we relax the test and allow not setting `http.status_code` or should we do something similar as done with servlet and set status code to `500` when there was an exception an status code was not set?